css/CSS2/floats/computed-float-position-absolute.html

```diff
@@ -0,0 +1,30 @@
+<!doctype html>
+<link rel="help" href="https://drafts.csswg.org/css2/visuren.html" />
+<title>The computed value of float with absolute positioning when there is no box should be "none"</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<style>
+
+div[id] {
+	position: absolute;
+	float: left;
+}
+
+</style>
+
+<div id="test1" style="display: none"></div>
+<div id="test2" style="display: contents"></div>
+<div style="display: none"><div id="test3"></div></div>
+
+<script>
+var test_description = document.title;
+promise_test(
+	t => {
+		return new Promise(test => addEventListener('load', e=>test()))
```

Comment:
There's no need for the test to wait for the load event; this can be a normal test().

Reply:
I guess, but the test is autogenerated, and it is a generally impossible to resolve problem to decide that there won't be dependencies like font loading that are required in the general case; is there a problem if the test loads for the load event? I wouldn't assume this to change test execution time given there isn't anything load will be waiting on in this case.